digiwf-integrations/digiwf-openai-integration/digiwf-openai-integration-core/src/main/java/de/muenchen/oss/digiwf/openai/integration/adapter/in/streaming/MessageProcessor.java

```diff
@@ -0,0 +1,102 @@
+package de.muenchen.oss.digiwf.openai.integration.adapter.in.streaming;
+
+import de.muenchen.oss.digiwf.message.process.api.error.BpmnError;
+import de.muenchen.oss.digiwf.message.process.api.error.IncidentError;
+import de.muenchen.oss.digiwf.openai.integration.adapter.in.streaming.dto.ClassifyDto;
+import de.muenchen.oss.digiwf.openai.integration.adapter.in.streaming.dto.ExtractDataDto;
+import de.muenchen.oss.digiwf.openai.integration.adapter.in.streaming.dto.GenerateMailDto;
+import de.muenchen.oss.digiwf.openai.integration.adapter.in.streaming.dto.PromptDto;
+import de.muenchen.oss.digiwf.openai.integration.adapter.in.streaming.dto.SummarizeDto;
+import de.muenchen.oss.digiwf.openai.integration.adapter.in.streaming.dto.TranslateDto;
+import de.muenchen.oss.digiwf.openai.integration.adapter.out.dto.ChatRequest;
+import de.muenchen.oss.digiwf.openai.integration.adapter.out.dto.ClassifyRequest;
+import de.muenchen.oss.digiwf.openai.integration.adapter.out.dto.ExtractDataRequest;
+import de.muenchen.oss.digiwf.openai.integration.adapter.out.dto.GenerateMailRequest;
+import de.muenchen.oss.digiwf.openai.integration.adapter.out.dto.OpenAiResponse;
+import de.muenchen.oss.digiwf.openai.integration.adapter.out.dto.SummarizeRequest;
+import de.muenchen.oss.digiwf.openai.integration.adapter.out.dto.TranslateRequest;
+import de.muenchen.oss.digiwf.openai.integration.application.port.in.OpenAiInPort;
+import de.muenchen.oss.digiwf.openai.integration.application.port.out.IntegrationOutPort;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.messaging.Message;
+
+import java.util.Map;
+import java.util.function.Consumer;
+
+@RequiredArgsConstructor
+@Slf4j
+public class MessageProcessor {
+
+    private final OpenAiInPort openAiInPort;
+    private final IntegrationOutPort integrationOutPort;
```

Comment:
Hier dürfen nur InPorts verwendet werden.

Reply:
Ist aktuell nicht der Fall in den anderen Int. Konzept wird in #1465 ausgearbeitet